### DISCUSSION TITLE
Is there still a way for the slider body to be transparent after the last update?

### DISCUSSION THREAD

**Discussion by minasminas** [ppy/osu#14296] [Q&A] [answered]
My skin uses a "SliderTrackOverride: 0,0,0,0" value, which was transparent on lazer up to the last patch. I think [this](https://github.com/ppy/osu/pull/14205) PR changed the behaviour of my skin to be like in stable. I guess this is generally a good thing, but I would love to know if there is some new way to make the slider body transparent, as I liked it a lot more this way.

**Comment by bdach** [comment, marked as answer]
No, there is not - at least not for now. It was never intended to add support for transparent body colours to legacy skins. This will very likely be supported however by a future skinning system.

**Reply by minasminas** [reply]
Thanks for the answer, even tho it's a little sad for me. Looking forward to new ways of hopefully doing that in the future!

**Reply by TheKevinGDX** [reply]
ok, yo realmente deseo que esto regrese en un futuro no muy lejano

**Reply by frenzibyte** [reply]
1. English please
2. "This will very likely be supported however by a future skinning system."
